mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -455,6 +455,8 @@ trait Completions { this: MetalsGlobal =>
     latestEnclosingArg match {
       case MillIvyExtractor(dep) =>
         MillIvyCompletion(pos, text, dep)
+      case SbtLibExtractor(pos, dep) if pos.source.path.isSbt =>
```

Comment:
We can also have Scala targets within sbt, but we can handle it separately.

We would need to add an additional flag `isSbt` for PresentationCompile interface or within PresentationCompilerConfig, which might be better.